common/src/warp_runner/manager/commands/constellation_commands.rs

```diff
@@ -299,7 +299,13 @@ fn get_items_from_current_directory(
 }
 
 fn get_directories_opened() -> Vec<Directory> {
-    DIRECTORIES_AVAILABLE_TO_BROWSE.read().to_owned()
+    let mut dirs_opened = DIRECTORIES_AVAILABLE_TO_BROWSE.read().clone();
+    if let Some(first_dir) = dirs_opened.first() {
```

Comment:
@lgmarchi do you know where the directory named "un-named directory" comes from? I added a logging statement where `dirs_opened.remove(0)` was called and that's the name of the directory getting removed.